src/platform/forms/save-in-progress/RoutedSavableApp.jsx

```diff
@@ -51,15 +51,21 @@ class RoutedSavableApp extends React.Component {
     //  saved form / prefill
     // If we're in production, we'll redirect if we start in the middle of a form
     // In development, we won't redirect unless we append the URL with `?redirect`
-    const { currentLocation } = this.props;
+    const { currentLocation, formConfig } = this.props;
+    const { additionalRoutes = [] } = formConfig;
+    const additionalSafePaths =
+      additionalRoutes && additionalRoutes.map(route => route.path);
     const trimmedPathname = currentLocation.pathname.replace(/\/$/, '');
     const resumeForm = trimmedPathname.endsWith('resume');
     const devRedirect =
       (!environment.isLocalhost() &&
```

Comment:
This check explains why staging was behaving differently from local testing. When testing locally I removed this check and was able to see to same behavior locally as staging.